bin/MadGraph5_aMCatNLO/cards/production/13TeV/ZJets_LO_MLM/ZJetsToNuNu_HT-400to600/ZJetsToNuNu_HT400to600_run_card.dat

```diff
@@ -0,0 +1,255 @@
+# Number of helicities to sum per event (0 = all helicities)
+# 0 gives more stable result, but longer run time (needed for
+# long decay chains e.g.).
+# Use >=2 if most helicities contribute, e.g. pure QCD.
+#*************************************************************
+   0  = nhel          ! Number of helicities used per event
+#*******************                                                 
+# Standard Cuts
+#*******************                                                 
+#                                                                    
+#*********************************************************************
+# Minimum and maximum pt's (for max, -1 means no cut)                *
+#*********************************************************************
+<<<<<<< HEAD
+ 0.01  = ptj       ! minimum pt for the jets 
+=======
+ 10  = ptj       ! minimum pt for the jets 
+>>>>>>> f126f7f291d8749c0dd4fba078265712b1784185
+  0  = ptb       ! minimum pt for the b 
+ 10  = pta       ! minimum pt for the photons 
+ 0  = ptl       ! minimum pt for the charged leptons 
+  0  = misset    ! minimum missing Et (sum of neutrino's momenta)
+  0  = ptheavy   ! minimum pt for one heavy final state
+ 1.0 = ptonium   ! minimum pt for the quarkonium states
+ 1d5  = ptjmax    ! maximum pt for the jets
+ 1d5  = ptbmax    ! maximum pt for the b
+ 1d5  = ptamax    ! maximum pt for the photons
+ 1d5  = ptlmax    ! maximum pt for the charged leptons
+ 1d5  = missetmax ! maximum missing Et (sum of neutrino's momenta)
+#*********************************************************************
+# Minimum and maximum E's (in the lab frame)                         *
+#*********************************************************************
+  0  = ej     ! minimum E for the jets 
+  0  = eb     ! minimum E for the b 
+  0  = ea     ! minimum E for the photons 
+  0  = el     ! minimum E for the charged leptons 
+ 1d5   = ejmax ! maximum E for the jets
+ 1d5   = ebmax ! maximum E for the b
+ 1d5   = eamax ! maximum E for the photons
+ 1d5   = elmax ! maximum E for the charged leptons
+#*********************************************************************
+# Maximum and minimum absolute rapidity (for max, -1 means no cut)   *
+#*********************************************************************
+   5  = etaj    ! max rap for the jets 
+  1d2  = etab    ! max rap for the b
+ 2.5  = etaa    ! max rap for the photons 
+ 1d2  = etal    ! max rap for the charged leptons 
+ 0.6  = etaonium ! max rap for the quarkonium states
+   0  = etajmin ! min rap for the jets
+   0  = etabmin ! min rap for the b
+   0  = etaamin ! min rap for the photons
+   0  = etalmin ! main rap for the charged leptons
+#*********************************************************************
+# Minimum and maximum DeltaR distance                                *
+#*********************************************************************
+<<<<<<< HEAD
```

Comment:
The conflict probably entered the file at an earlier stage, when performing some git commands in one of the previous areas.  (Ie these conflict resolution lines are present in the files which you've copied into the clean directory)

You should be able to simply fix them and add a commit.